crawler.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import absolute_import
+
+import os
+import time
+
+import boto3
+from six.moves import range
+
+from automation import CommandSequence, TaskManager
+from automation.utilities import rediswq
+from test.utilities import LocalS3Session, local_s3_bucket
+
+# Configuration via environment variables
+NUM_BROWSERS = int(os.getenv('NUM_BROWSERS', '1'))
+REDIS_QUEUE_NAME = os.getenv('REDIS_QUEUE_NAME', 'crawl-queue')
+CRAWL_DIRECTORY = os.getenv('CRAWL_DIRECTORY', 'crawl-data')
+S3_BUCKET = os.getenv('S3_BUCKET', 'openwpm-crawls')
+HTTP_INSTRUMENT = os.getenv('HTTP_INSTRUMENT', '1') == '1'
```

Comment:
I'm not sure if it makes sense to pass this level of configuration in the yaml file. My thought for the general workflow:

1. A user closes the repo

Reply:
TLDR: Keeping high-level configuration allows for crawls to be made without being familiar with crawl engineering.

Long version:
That sounds like the advanced/customized workflow to me, in which maintaining a separate crawler.py file makes a lot of sense (simple Dockerfile: `FROM openwpm; ADD crawler.py`, then push), but I imagine the first-timer/general crawl workflow be something like:

1. They launch a simple crawl with their desired configuration (or the default configuration) via kubernetes (either locally or on GCP) and specify the set of sites the enqueue in redis and the name for the crawl.

Ie, using the stock openwpm image, which has been pushed to the docker registry by CI.

After performing one or many such crawls (for testing or production purposes), it may be of interest to tweak one or many aspects of the crawl strategy, at which point it makes sense going through the advanced/customized workflow above. 

I am even open to including configuration options such as `CRAWL_SEED_LIST="alexatop1k"`, `CRAWL_STRATEGY="stateless"`, `LINK_FOLLOWING_STRATEGY="random", `LINK_FOLLOWING_DEPTH="1"` etc that would allow new crawls and replication of previous studies / crawls with even more ease.

Reply:
Makes sense; thanks for the response.